tools/pytorch2onnx.py

```
@@ -1,124 +1,168 @@
 import argparse
-import io
+import os.path as osp
+from functools import partial
 
 import mmcv
+import numpy as np
 import onnx
+import onnxruntime as rt
 import torch
-from mmcv.ops import RoIAlign, RoIPool
+from mmcv.onnx.symbolic import register_extra_symbolics
```

We can raise an error message if mmcv version is low.